<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="UXCPS-PC90" name="Test_1" tests="2" failures="1" timestamp="21 Oct 2016 05:22:18 GMT" time="13.379" errors="0">
  <testcase name="myTest1" time="2.463" classname="com.learnautomation.selenium.Demo1"/>
  <testcase name="myTest2" time="3.618" classname="com.learnautomation.selenium.Demo1">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:92)
	at org.junit.Assert.assertTrue(Assert.java:43)
	at org.junit.Assert.assertTrue(Assert.java:54)
	at com.learnautomation.selenium.Demo1.myTest2(Demo1.java:51)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- myTest2 -->
</testsuite> <!-- Test_1 -->
